MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -473,65 +470,56 @@ private StreamInfo BuildAudioItem(MediaSourceInfo item, AudioOptions options)
             return options.GetMaxBitrate(isAudio);
         }
 
-        private Tuple<List<PlayMethod>, List<TranscodeReason>> GetAudioDirectPlayMethods(MediaSourceInfo item, MediaStream audioStream, AudioOptions options)
+        private (IEnumerable<PlayMethod>, IEnumerable<TranscodeReason>) GetAudioDirectPlayMethods(MediaSourceInfo item, MediaStream audioStream, AudioOptions options)
         {
-            var transcodeReasons = new List<TranscodeReason>();
+            DirectPlayProfile directPlayProfile = options.Profile.DirectPlayProfiles
+                .FirstOrDefault(x => x.Type == DlnaProfileType.Audio && IsAudioDirectPlaySupported(x, item, audioStream));
 
-            DirectPlayProfile directPlayProfile = null;
-            foreach (DirectPlayProfile i in options.Profile.DirectPlayProfiles)
+            if (directPlayProfile == null)
```

Comment:
I'm very confused by this if. Does the device need a directplayprofile to direct stream?

Reply:
This funtion (`GetAudioDirectPlayMethods`) checks if it can direct play, not stream.